build/ci/release.yml

```diff
@@ -189,14 +189,13 @@ functions:
           - revision
           - created_at
           - ext
-          - signing_auth_token_44
-          - signing_auth_token_46
```

Comment:
I think we use this for the QA repo

Reply:
what do you mean by QA repo? do you see any pointers in the code?

Reply:
for snapshots it goes to a testing repo like https://repo.mongodb.com/apt/ubuntu/dists/jammy/mongodb-enterprise/development/ the logic used to be by the key being anything but a stable version if I remember correctly

Reply:
I've added it back but so far I can't find a pointer to 4.6